pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -163,11 +163,22 @@ func (cc *cadvisorClient) MachineInfo() (*cadvisorapi.MachineInfo, error) {
 	return cc.GetMachineInfo()
 }
 
-func (cc *cadvisorClient) DockerImagesFsInfo() (cadvisorapiv2.FsInfo, error) {
-	return cc.getFsInfo(cadvisorfs.LabelDockerImages)
+func (cc *cadvisorClient) ImagesFsInfo(runtime string) (cadvisorapiv2.FsInfo, error) {
+	var label string
+
+	switch runtime {
+	case "docker":
+		label = cadvisorfs.LabelDockerImages
+	case "rkt":
+		label = cadvisorfs.LabelRktImages
+	default:
+		return cadvisorapiv2.FsInfo{}, fmt.Errorf("ImagesFsInfo: unknown runtime: %v", runtime)
+	}
+
+	return cc.getFsInfo(label)
 }
 
-func (cc *cadvisorClient) RootFsInfo() (cadvisorapiv2.FsInfo, error) {
+func (cc *cadvisorClient) RootFsInfo(_ string) (cadvisorapiv2.FsInfo, error) {
```

Comment:
We used to have DockerImagesFsInfo() and RootFsInfo()

both were used as arguments to a function.  DockerImagesFsInfo was renamed to ImagesFsInfo(runtime string) so in order to not duplicate code, modified RootFsInfo() to keep the same prototype as it